### PR TITLE
Review of deployment guide for block storage (SOC-8981)

### PR DESCRIPTION
**Pull request by asettle**
WIP for block storage content.

### CODE REVIEW

**Comment by asettle** [comment]
Due to the changes in the product teams, the WIP we were waiting on no longer is required. This PR can be reviewed accordingly.

**Comment by asettle** [comment]
Hey @guangyee - would appreciate your help reviewing some of these PRs if you have time. It was originally marked WIP as we were waiting on TR's word on what we officially support, but TR emailed Walter Boring and said we didn't have an official word on what we supported. At this point in time, I'm not sure how relevant it is. Advice appreciated.

**Comment by KeithMnemonic** [comment]
the content about the volume type is correct. I thought for the CLM docs we had all of this but maybe it was not clear what to copy or maybe this was for Crowbar

**Comment by asettle** [comment]
> the content about the volume type is correct. I thought for the CLM docs we had all of this but maybe it was not clear what to copy or maybe this was for Crowbar

Could you comment in the doc so I know exactly where you're referring to? I'm going off everything Walter Boring provided before he left.

File: xml/installation-kvm_xpointer.xml [hunk 1]
```diff
@@ -798,12 +798,65 @@ ansible-playbook -i hosts/localhost ready-deployment.yml</screen>
...
+      </para>
+      <screen>
+        openstack volume type create --public NAME
+      </screen>
+    </step>
+    <step>
+      <para>
+        Volume types do not have to be tied to a backend. They can contain
+        attributes that can be applied to a volume during creation. One of
+        those attributes is the <literal>volume_backend_name</literal>. By settings
```
Comment (KeithMnemonic) [inline xml/installation-kvm_xpointer.xml]: s/settings/setting

File: xml/installation-kvm_xpointer.xml [hunk 2]
```diff
@@ -798,12 +798,65 @@ ansible-playbook -i hosts/localhost ready-deployment.yml</screen>
...
+       Create a volume type:
+      </para>
+      <screen>
+        openstack volume type create --public NAME
+      </screen>
+    </step>
+    <step>
+      <para>
+        Volume types do not have to be tied to a backend. They can contain
+        attributes that can be applied to a volume during creation. One of
```
Comment (KeithMnemonic) [inline xml/installation-kvm_xpointer.xml]: these are referred to as "extra specs"

**Pull request merged**